src/Dfc.CourseDirectory.Core/DataStore/IRegionCache.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core.Models;
+
+namespace Dfc.CourseDirectory.Core.DataStore
```

Comment:
Can we put this in an appropriate namespace? `Core.Caching`? `Core.Data.Regions`?

Reply:
Further thoughts....

The abstraction wants to be independent from the implementation. So far as `IRegionCache` lives in a generalised namespace, and `RegionCache` should become `SqlRegionCache` and be an Sql specific implementation of `IRegionCache`.